packages/dx-react-grid/src/plugins/table-header-row.tsx

```diff
@@ -94,7 +92,7 @@ class TableHeaderRowBase extends React.PureComponent<TableHeaderRowProps> {
                     draggingEnabled={draggingEnabled && atLeastOneDataColumn}
                     resizingEnabled={
                       tableColumnResizingEnabled
-                      && (!lastColumn || columnResizingMode === 'widget')
+                      && (!!nextColumnName || columnResizingMode === 'widget')
```

Comment:
Note: very wired using `widget` as a sting in a lot of places instead of using one constant.